widget/embedded/src/components/TokenList/TokenList.tsx

```diff
@@ -220,7 +217,9 @@ export function TokenList(props: PropTypes) {
                     : token.name || undefined
                 }
                 end={
-                  loadingWallet ? (
+                  action ? (
```

Comment:
I'm against of nested ternaries. You can do something like:

```

        const endRenderer = () => {
          if (action) {
            return action(token);
          }
          if (loadingWallet) {
            return (
              <End>
                <Skeleton variant="text" size="large" width={70} />
                <Divider size={4} />
                <Skeleton variant="text" size="medium" width={50} />
              </End>
            );
          }

          return (
            tokenBalance && (
              <BalanceContainer>
                <TokenBalance variant="title" size="small">
                  {tokenBalance.amount}
                </TokenBalance>
                <div />
                {tokenBalance.usdValue && (
                  <Typography
                    variant="body"
                    className={usdValueStyles()}
                    size="xsmall">
                    {`$${tokenBalance.usdValue}`}
                  </Typography>
                )}
              </BalanceContainer>
            )
          );
        };
        
```

then 
```
end={endRenderer()}
```

Reply:
okay, I added endRenderer to token list component